app/controllers/pages_controller.rb

```diff
@@ -3,27 +3,55 @@
 class PagesController < ApplicationController
   layout 'errors_and_contact'
 
-  def help; end
+  def help
+    if current_user.present?
+      @user = Nomis::Elite2::UserApi.user_details(current_user)
+      @contact = ContactSubmission.new(email_address: @user.email_address.first,
+                                       name: @user.full_name_ordered,
+                                       prison: PrisonService.name_for(
+                                         @user.active_case_load_id)
+      )
+    else
+      @contact = ContactSubmission.new
+    end
+  end
+
+  def create_help
+    @user = Nomis::Elite2::UserApi.user_details(current_user) if current_user.present?
```

Comment:
What is this used for?

Reply:
To create the actual help form and trigger the creation of the Zendesk ticket if it's correctly formatted i.e. has all the right information filled in